Elbow Saver/Models/ExerciseTimerViewModel.swift

```diff
@@ -0,0 +1,138 @@
+//
+//  ExerciseTimerViewModel.swift
+//  ExerciseTimerViewModel
+//
+//  Created by Daniel Luo on 8/23/21.
+//
+
+import Foundation
+
+class ExerciseTimerViewModel: ObservableObject {
+    
+    // Fixed values, not user-adjustable
+    /// The number of eccentric repetitions per set of exercise.
+    static let repsPerSet: Int = 3
+    /// The duration over which each rep is to be performed.
+    static let secondsPerRep: Int = 4
+    /// The duration between reps during which user returns to the exercise start position
+    static let secondsBetweenReps: Int = 1
+    
+    /// Settings for session including total number of sets, rest period between sets.
+    let sessionSettings: ExerciseSessionSettings
```

Comment:
If we're being pedantic, the viewModel shouldn't expose the model directly to the view. The vars needed by the view should be accessed via computed properties. And the model should be private

```
var someProp: Type { sessionSettings.sameProp }
```

Reply:
Done